apps/root/src/common/components/address/index.tsx

```diff
@@ -25,6 +27,7 @@ const Address = ({ address, trimAddress, trimSize }: AddressProps) => {
   return (
     <>
       {addressEns ||
+        storedLabels[address] ||
```

Comment:
We should prioritize the user stored label to the ens name

Reply:
Sure! Done 😄